src/driver/amdxdna/amdxdna_drv.c

```diff
@@ -263,6 +272,7 @@ amdxdna_get_dev_info(struct pci_dev *pdev)
 static int amdxdna_probe(struct pci_dev *pdev, const struct pci_device_id *id)
 {
 	struct amdxdna_dev *xdna;
+	struct device *dev = &pdev->dev;
```

Comment:
```suggestion
	struct device *dev = &pdev->dev;
	struct amdxdna_dev *xdna;
```
Reverse XMAS tree declarations please.